docs/install.md

```diff
@@ -33,12 +33,32 @@ For credential-based logins such as Azure Container Registry, Docker Hub and Qua
 $ echo '{"username":"jdoe","password":"secret","email":"jdoe@acme.com"}' | base64
 ```
 
-For token-based logins such as Google Container Registry and Amazon ECR, use:
+For token-based logins including Amazon ECR, use:
 
 ```
 $ echo '{"registrytoken":"9cbaf023786cd7"}' | base64
 ```
 
+### For Google Container Registry:
```

Comment:
Unfortunately, I'm not going to be able to accept the CLA.

Unless I hear otherwise, I'll revert this tomorrow morning.

Apologies for the inconvenience.